Makefile

```diff
@@ -37,6 +39,9 @@ build-release:
 	# make sure to always set version with vX.X.X
 	docker build --pull --no-cache --build-arg IS_RELEASE=true --build-arg ZEND_MAINTAINER_KEYS="${ZEND_MAINTAINER_KEYS}" --build-arg ZEN_COMMITTISH=${ZEN_COMMITTISH} -t rosetta-zen:$(version) .;
 	docker save rosetta-zen:$(version) | ${GZIP_CMD} > rosetta-zen-$(version).tar.gz;
+	${DOCKER_LOGIN_CMD};
+	docker push "${DOCKER_IMAGE_NAME}:latest";
```

Comment:
I don't think the image will be tagged as latest at this point, and it won't be named zencash/rosetta-zen. Add `docker tag` commands.